core-3/src/main/java/scanning/config/ProjectConfigScan.java

```diff
@@ -0,0 +1,11 @@
+package scanning.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackages = ("scanning"))
+public class ProjectConfigScan {
```

Comment:
remove this configuration bean, this context should be loaded with scanning feature